samples/telemetry/debug.json

```diff
@@ -1,16 +1,11 @@
 {
   "_dd": {
-    "event_type": "internal_telemetry"
+    "format_version": 2
   },
+  "type": "telemetry",
   "date": 1591284175342,
   "service": "sample-sdk",
   "version": "1.2.3",
-  "status": "debug",
-  "message": "Session inconsistencies detected",
-  "debug": {
-    "cookie": "foo",
-    "session": "bar"
-  },
   "application": {
```

Comment:
Those fields should not be duplicated within the event.

Reply:
It depends on whether we want to index those fields. Only what's in the `telemetry` will be indexed. Keeping these fields on the root is interesting for the processing phase.
I see 2 options. Either the SDK duplicates those fields, which might be easy since you know which ones exist (?). Or, the backend copies the fields but has to check for existence.

Reply:
In this case why `action` field is not duplicated in the `Telemetry` object (and in `Telemetry.Error` in case of error event)?

Reply:
This feels strange to me. This implies that some part of the code has only access to the `telemetry` field, and some other part won't have access to the `telemetry` field? Is there a reason for that? Why not use the whole event everywhere?

Reply:
Discussed it with the team: I removed the duplication, and the fields will stay at the root.
Answering your question, the RUM intake will ingest this event. We would keep some consistency between schemas, thus the metadata around the `telemetry` field. However, the logs intake is less restrictive and we can index the `telemetry` field containing the logs event only.
I pushed another commit.